src/client/Microsoft.Identity.Client/AppConfig/CacheOptions.cs

```diff
@@ -34,9 +36,16 @@ public CacheOptions()
         /// Constructor
         /// </summary>
         /// <param name="useSharedCache">Set to true to share the cache between all ClientApplication objects. The cache becomes static. <see cref="UseSharedCache"/> for a detailed description. </param>
-        public CacheOptions(bool useSharedCache)
+        /// <param name="accessTokenExpirationScanFrequency">Sets to non-null to enable expiration of access tokens. <see cref="AccessTokenExpirationScanFrequency"/> for a detailed description. </param>
+        public CacheOptions(bool useSharedCache, TimeSpan? accessTokenExpirationScanFrequency = null)
         {
+            if (accessTokenExpirationScanFrequency.HasValue && accessTokenExpirationScanFrequency.Value <= TimeSpan.Zero)
+            {
+                throw new ArgumentException("accessTokenExpirationScanFrequency must be a positive TimeSpan", nameof(accessTokenExpirationScanFrequency));
+            }
+
             UseSharedCache = useSharedCache;
+            AccessTokenExpirationScanFrequency = accessTokenExpirationScanFrequency;
```

Comment:
@bgavrilMS does it make sense to set this to a default value to avoid caches expanding indefinitely? Also, is there a recommended value for this?